app/_src/gateway/support/index.md

```diff
@@ -45,6 +45,9 @@ Customers with platinum or higher subscriptions may request fixes outside of the
 Kong supports the following versions of {{site.ee_product_name}}: 
 
 {% navtabs %}
+  {% navtab 3.3 %}
```

Comment:
Can you make the same change to the file `support/third-party.md` and `support/browser.md`?